qa/zcash/performance-measurements.sh

```diff
@@ -406,6 +418,73 @@ case "$1" in
                 exit 1
         esac
         ;;
+    heaptrack)
+        zcashd_heaptrack_start
+        case "$2" in
+            sleep)
+                zcash_rpc zcbenchmark sleep 1
+                ;;
+            parameterloading)
+                zcash_rpc zcbenchmark parameterloading 1
+                ;;
+            createsaplingspend)
+                zcash_rpc zcbenchmark createsaplingspend 1
+                ;;
+            verifysaplingspend)
+                zcash_rpc zcbenchmark verifysaplingspend 1
+                ;;
+            createsaplingoutput)
+                zcash_rpc zcbenchmark createsaplingoutput 1
+                ;;
+            verifysaplingoutput)
+                zcash_rpc zcbenchmark verifysaplingoutput 1
+                ;;
+            createjoinsplit)
+                zcash_rpc_veryslow zcbenchmark createjoinsplit 1 "${@:3}"
+                ;;
+            verifyjoinsplit)
+                zcash_rpc zcbenchmark verifyjoinsplit 1 "\"$RAWJOINSPLIT\""
+                ;;
+            solveequihash)
+                zcash_rpc_veryslow zcbenchmark solveequihash 1 "${@:3}"
+                ;;
+            verifyequihash)
+                zcash_rpc zcbenchmark verifyequihash 1
+                ;;
+            trydecryptnotes)
+                zcash_rpc zcbenchmark trydecryptnotes 1 "${@:3}"
+                ;;
+            incnotewitnesses)
+                zcash_rpc zcbenchmark incnotewitnesses 1 "${@:3}"
+                ;;
+            connectblockslow)
+                extract_benchmark_data_107134
+                zcash_rpc zcbenchmark connectblockslow 1
+                ;;
+            connectblocksapling)
+                extract_benchmark_data_1723244
+                zcash_rpc zcbenchmark connectblocksapling 1
+                ;;
+            connectblockorchard)
+                extract_benchmark_data_1708048
+                zcash_rpc zcbenchmark connectblockorchard 1
+                ;;
+            sendtoaddress)
+                zcash_rpc zcbenchmark sendtoaddress 1 "${@:4}"
+                ;;
+            loadwallet)
+                # The initial load is sufficient for measurement
+                ;;
+            listunspent)
+                zcash_rpc zcbenchmark listunspent 1
+                ;;
+            *)
+                zcashd_heaptrack_stop
+                echo "Bad arguments to heaptrack."
+                exit 1
+        esac
+        zcashd_heaptrack_stop
+        ;;
```

Comment:
Similarly, these are now all unnecessary, as we can just use the `memory` argument.
```suggestion
```